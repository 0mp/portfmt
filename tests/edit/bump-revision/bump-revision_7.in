PORTREVISION=	2 #muh
