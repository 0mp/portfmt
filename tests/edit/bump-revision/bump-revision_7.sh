${PORTEDIT} bump-revision bump-revision_7.in | \
	diff -L bump-revision_7.expected -L bump-revision_7.actual \
		-u bump-revision_7.expected -
