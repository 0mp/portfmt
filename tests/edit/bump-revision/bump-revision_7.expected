PORTREVISION=	3 #muh
